console/console-init/ui/src/Components/AddressSpace/AddressSpaceHeader.css

```diff
@@ -0,0 +1,12 @@
+.l_split_m_gutter_MarginRight {
```

Comment:
This `l` namespace is too generic, these classes should be specific to the `AddressSpaceHeader` component